src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -72,18 +67,13 @@ public async Task InvokeAsync(HttpContext context)
                         gqlRequest = deserializationResult.Single;
                         gqlBatchRequest = deserializationResult.Batch;
                         break;
-
-                    case GraphQLContentType:
-                        gqlRequest.Query = await ReadAsStringAsync(httpRequest.Body).ConfigureAwait(false);
-                        break;
-
-                    case FormUrlEncodedContentType:
```

Comment:
Where is this case now?

Reply:
It's no longer recommended in the serving over HTTP documentation, so I don't think we should bother. Writing a test for it was a pain in the butt.

Reply:
I would prefer to keep this feature supported. This PR is already very large and we are solving a different problem. You might consider removing `application/x-www-form-urlencoded` support in  separate PR.

> Writing a test for it was a pain in the butt.

You may not to do tests for this.

Reply:
Alright, I'll bring it back.